sumpy/e2e.py

```diff
@@ -48,7 +48,9 @@
 
 # {{{ translation base class
 
-class E2EBase(KernelCacheWrapper):
+class E2EBase(KernelCacheMixin):
```

Comment:
`P2PBase` and some others also inherit from `KernelComputation`. Are these missing that on purpose?

Reply:
Yep, they take different arguments in init and everything!